Gemfile

```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 
 gemspec
 
-gem "activerecord", '~> 5.2.4.5'
+gem "activerecord", "~> 6.0.3"
```

Comment:
of note, for these we want to limit to 3 digits. the last digit is allowed to change.
So pegging to 5.2.4.5 will allow 5.2.4.6 but not 5.2.5.